app/src/mocks/react-native-passport-reader.ts

```diff
@@ -6,17 +6,7 @@
 
 // Mock PassportReader object with analytics methods
 export const PassportReader = {
-  configure: (
```

Comment:
missed this ai hallucination....created an unusable typing that doesn't exist anywhere that broke nfc scanning

@seshanthS